ReactWindows/ReactNative.Shared/UIManager/UIViewOperationQueue.cs

```diff
@@ -27,6 +27,7 @@ public class UIViewOperationQueue
         private const long MaxNonBatchedTicksPerFrame = 83333;
 
         private static Stopwatch s_stopwatch = Stopwatch.StartNew();
+        private static readonly Action[] EMPTY_ACTIONS_ARRAY = new Action[0];
 
```

Comment:
Isn't it create a new Array btw?
It's a bad idea in the general to allocate on the UI thread.